src/frontend/scss/components/templates/courses/cms/_course_detail.scss

```diff
@@ -151,6 +179,30 @@
         margin-top: 1rem;
       }
     }
+
+    // Add CSS style only to edit mode
+    &__hidden.cms-render-model {
+      &::before {
+        background-image: r-theme-val(course-detail, run-catalog-visibility-hidden-logo);
+        content: '';
+        background-size: 1rem 1rem;
+        display: inline-block;
+        width: 1rem;
+        height: 1rem;
+        margin-left: -1.2rem;
+      }
+    }
+    &__course_only.cms-render-model {
+      &::before {
+        background-image: r-theme-val(course-detail, run-catalog-visibility-course-only-logo);
+        content: '';
+        background-size: 1rem 1rem;
+        display: inline-block;
+        width: 1rem;
+        height: 1rem;
+        margin-left: -1.2rem;
+      }
+    }
   }
```

Comment:
Ditto.

Group common styles and use BEM modifier syntax `--` instead of `__`

Reply:
Changed.